util/dvsim/Deploy.py

```diff
@@ -463,6 +465,15 @@ def pre_launch(self):
         self.launcher.renew_odir = True
 
     def post_finish(self, status):
+        try:
+            self.job_runtime, self.simulated_time = get_runtimes(self.get_log_path(),
+                                                                 self.sim_cfg.tool)
```

Comment:
I wonder if `Launcher::_check_status()` is a better place to call this, given that it already opens the log file and reads the text into a list of lines. Somewhere here maybe:
https://github.com/lowRISC/opentitan/blob/106487eba9126c1d7cd0227502c625df29b55813/util/dvsim/Launcher.py#L284

That way, `get_runtimes` could accept `log_text: list` as a list of lines directly. File ops are indeed expensive, and re-opening and closing the same file twice for 30k tests can potentially slow down dvsim.

Reply:
Thanks Sri for the suggestion! I did put it there in the beginning of this PR, but I think I misunderstand your suggestion about calling `get_runtimes` after pass and fail are done. Yeah agree that this might slow down dvsim!